plugins/UsersManager/angularjs/manage-users/manage-users.controller.js

```diff
@@ -176,15 +186,32 @@
             }});
         };
 
+        this.regenerateUserTokenAuth = function (userLogin) {
+            var parameters = { userLogin: userLogin };
```

Comment:
The manage users integration is nice 👍  Be only good to have here a confirm/alert message as well similar to in "personal settings". Just in case people click on it accidentally etc.